src/App.js

```diff
@@ -1,25 +1,23 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import TodoCard from './components/Tasklist/TodoCard'
+import data from './components/data.json'
+import Header from './components/Header/Header'
+import styled from 'styled-components'
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <Main>
+      <Header />
+      {data.map(data => (
+        <TodoCard todo={data.todo} id={data.id} key={data.id} />
```

Comment:
NitPicker:
You dont need the id prop here as you dont use it atm in the ToDoCard